api/http/api_deployments.go

```diff
@@ -52,8 +52,9 @@ const (
 	// 15 minutes
 	DefaultDownloadLinkExpire = 15 * time.Minute
 	// 10 Mb
-	DefaultMaxMetaSize  = 1024 * 1024 * 10
-	DefaultMaxImageSize = 10 * 1024 * 1024 * 1024 // 10GiB
+	DefaultMaxMetaSize         = 1024 * 1024 * 10
+	DefaultMaxImageSize        = 10 * 1024 * 1024 * 1024 // 10GiB
+	DefaultMaxGenerateDataSize = 512 * 1024 * 1024       // 512MiB
```

Comment:
normally I would say: we should indicate that it is bytes in the identifier, but I see we dont do it here.